fuseml/questions.yaml

```diff
@@ -8,3 +8,24 @@ questions:
   type: string
   label: Path to custom repository with FuseML extensions
   group: "Global Options"
```

Comment:
Aren't there other options that should be featured in the global section ? I can think of at least of `systemDomain`.